src/spoke/VaultSpoke.sol

```diff
@@ -284,44 +288,34 @@ abstract contract VaultSpoke is IVaultSpoke, ERC20Upgradeable, NoncesKeyed, EIP7
 
   /// @inheritdoc IVaultSpoke
   function hub() public view returns (address) {
-    return address(_HUB);
+    return address(HUB);
```

Comment:
is this still necessary now ? (same for `assetId()`)

Reply:
i find it cleaner in this context purely bc of 'asset()' needed by 4626 standard

Reply:
definitely, which is why i had underscores earlier which all immutables internal which looked cleaner